.github/workflows/workflow.yaml

```diff
@@ -1,4 +1,5 @@
-name: Infrastructure integration workflow
+---
+name: Infra integration workflow
```

Comment:
Could you change the infra to howard. I'll update the repo name as mentioned in this [issue](https://github.com/ai-cfia/infra/issues/32)